manifest.md

```diff
@@ -5,6 +5,7 @@ The first goal is content-addressable images, by supporting an image model where
 The second goal is to allow multi-architecture images, through a "fat manifest" which references image manifests for platform-specific versions of an image.
 The third goal is to be translatable to the [OpenContainers/runtime-spec](https://github.com/opencontainers/runtime-spec)
 
+This section defines the `application/vnd.oci.image.manifest.list.v1+json` and `application/vnd.oci.image.manifest.v1+json` [media type](media-types.md).
```

Comment:
Can we break this up into two files?  Manifest lists and manifests seem as separate as manifests and configs.  And while I like the commits that clearly list the media types being defined, I think that change is orthogonal to the order-clarification.  Maybe it belongs in a separate PR?